src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -188,19 +180,17 @@ abstract contract IntegrationDeployer is ExistingDeploymentParser {
          * Running foundry like this will trigger the fork test profile,
          * lowering fuzz runs and using a remote RPC to test against mainnet state
          */
-        bool forkMainnet = 
-            _hash("forktest") ==
-            _hash(cheats.envOr(string("FOUNDRY_PROFILE"), string("default")));
+        bool forkMainnet = _hash("forktest") == _hash(cheats.envOr(string("FOUNDRY_PROFILE"), string("default")));
 
         if (forkMainnet) {
-            emit log("setUp: running tests against mainnet fork");
-            emit log_named_string("- using RPC url", cheats.rpcUrl("mainnet"));
-            emit log_named_uint("- forking at block", mainnetForkBlock);
+            console.log("Setting up `%s` integration tests:", "MAINNET_FORK".green().bold());
```

Comment:
qq: do we have to update the verbosity to display with emit vs console.log?

Reply:
I believe they are the same, will confirm though.

Reply:
they appear to work the same..

Reply:
Awesome, thanks for verifying